test/data/mock.php

```diff
@@ -215,6 +215,14 @@ protected function cspClean( $req ) {
 		file_put_contents( $this->cspFile, '' );
 		unlink( $this->cspFile );
 	}
+	
+	public function responseURL($req) {
+		if ( isset($req->query["url"]) ) {
+			header( "Location: $req->query[url]" );
```

Comment:
Does that also set the `302` status code, as code in `middleware-mockserver` does?

Reply:
Yes